.env.example

```
@@ -1,3 +1,3 @@
-VITE_SUPABASE_URL=
-VITE_SUPABASE_ANON_KEY=
-VITE_API_URL=
\ No newline at end of file
+SUPABASE_URL=https://hdosviedtzfribvnynij.supabase.co
+SUPABASE_ANON_KEY=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJzdXBhYmFzZSIsInJlZiI6Imhkb3N2aWVkdHpmcmlidm55bmlqIiwicm9sZSI6ImFub24iLCJpYXQiOjE3NzQzNTkwMjIsImV4cCI6MjA4OTkzNTAyMn0.ejNMQA60yilo6vNO0T_lAEKtNxwWExMCXB64AR-aphI
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The SUPABASE_ANON_KEY key should go before the SUPABASE_URL key

(UnorderedKey)